snuba/datasets/replays_processor.py

```diff
@@ -71,7 +71,7 @@ def _process_base_event_values(
         processed["replay_id"] = str(uuid.UUID(event_dict["replay_id"]))
         processed["project_id"] = event_dict["project_id"]
 
-        processed["sequence_id"] = event_dict["sequence_id"]
+        processed["segment_id"] = event_dict["segment_id"]
```

Comment:
You will need to write the event_hash column. Are you planning to do this here or as a follow up?

Reply:
followup, doing it in #2962

Reply:
Ok. Just keep in mind any replay data currently in this table will not be valid until then as rows will be incorrectly merged

Reply:
table is empty right now 👍🏼